data/hai/hai news.txt

```diff
@@ -0,0 +1,251 @@
+
+
+# Visual suggestions: scruffy and disgruntled; religious and relaxed; sharply-dressed and clean-shaven
+news "reformed pirate in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Reformed pirate"
+	portrait
+		portrait/human01
+		portrait/human09
+		portrait/human23
+		portrait/human24
+		portrait/human32
+		portrait/human34
+		portrait/human41
+		portrait/human45
+		portrait/human51
+		portrait/human54
+		portrait/human57
+		portrait/human67
+		portrait/human77
+		portrait/human80
+		portrait/human92
+		portrait/human95
+		portrait/human115
+		portrait/human121
+		portrait/human124
+		portrait/human126
+	message
+		word
+			"I used to rob people for a living, and I'd be in jail now if I stayed in human space. Here, they've given me a job, and I've been sober for six weeks. This has really turned my life around."
+			"I lived on Freedom before I came here, and I could get space weed, death sticks, stardust, and toad juice any time of day or night. Here, the best I can find is spicy acorns."
+			"I used to be a slave on a pirate ship. My fool of a captain decided to attack Hai space. The Hai freed me, and now I work in one of their factories."
+			"I tried smuggling some bad blue to Hai space, but they caught me right away. I don't know what scanners they use in those Hai ships, but it's a lot more effective than the Navy's toys."
+			"If you think the Hai are weak-willed fur-balls, try watching one of their fencing matches. I've never seen a human move as fast as those Hai."
+			"I always said I'd get out of piracy one day, but I know the navy would lock me up for sure. Here in Hai space, I've made a new start."
+			"Being in Hai space has taught me to be a better person."
+			"Some of my friends abuse the Hai's kindness. After all they've done for me, I'd never do that."
+			"Back home, I had to rob people just to get enough money for food. Here I never go hungry."
+			"Don't judge me. You have no idea what I had to go through before I got here."
+			"I haven't stabbed anyone in months. This is an easier life."
+			"Hai are so forgiving here. I got into a fight with one and instead of pressing charges, he had a long talk with me about my choices in life."
+			"I always said I'd rethink my life, and now I have. I'm a farmer, not a killer."
+			"I used to hate myself for all the terrible crimes I did in human space. Here, the Hai have given me a fresh start."
+			"You have to re-invent yourself sometimes. I used to be a killer on a pirate ship, now I'm a canner in Hai factory."
+			# Duplicated intentionally so pirates say it too:
+			"I hope nobody is smuggling Hai technology out of here. I don't want to think about what would happen if the pirates got their hands on that stuff."
+			"A Hai that lives near me went to sleep one night and woke up in a box on Prime. Isn't that weird?"
```

Comment:
Still what?

Reply:
Deleted.

The kid from Allhome that ended up in a box wasn't the only one. It is to set up lore for the Philosopher's Paradise. I see how this one looks bizarre and out-of-place though.